new-style.css

```diff
@@ -1,1140 +1,1144 @@
-    margin-bottom: 0.75em;
+  font-size: 4em;
+  color: #5bc6f9;
+  font-weight: 200;
+  text-align: start;
+  line-height: 1em;
+  margin-bottom: 0.75em;
 }
 
 .above-the-fold .new-btn {
-    font-size: 1em;
-    margin-top: 3em;
-    background-color: #5fcf90;
-    transition: background-color 0.3s;
-    box-shadow: 0 4px 7px rgba(95, 207, 144, 0.61);
-    border-radius: 10px;
-    border: 0;
-    color: #fff;
-    padding: 0.625em 2.375em;
-    cursor: pointer;
-    font-weight: 500;
+  font-size: 1em;
+  margin-top: 2em;
+  background-color: #5fcf90;
+  transition: background-color 0.3s;
+  box-shadow: 0 4px 7px rgba(95, 207, 144, 0.61);
+  border-radius: 10px;
+  border: 0;
+  color: #fff;
+  padding: 0.625em 2.375em;
+  cursor: pointer;
+  font-weight: 500;
 }
 
 .above-the-fold .new-btn:hover {
-    background-color: #59c186;
+  background-color: #59c186;
 }
 
 .above-the-fold .new-subtitle {
-    font-size: 0.75em;
-    margin-top: 0.875em;
+  font-size: 0.75em;
+  margin-top: 1.5em;
 }
 
 .above-the-fold .graphic {
-    z-index: 1;
-    display: flex;
-    align-items: flex-end;
-    justify-content: end;
+  z-index: 1;
+  display: flex;
+  align-items: flex-end;
+  justify-content: end;
 }
 
 .above-the-fold .graphic img {
-    width: 100%;
-    filter: drop-shadow(3px 0 8px rgba(0, 0, 0, 0.33));
+  width: 100%;
+  filter: drop-shadow(3px 0 8px rgba(0, 0, 0, 0.33));
 }
 
 /* Scrolled default style */
 
 .header.scrolled {
-    position: fixed;
-    top: 0;
-    left: 0;
-    width: 100%;
-    padding-top: 1.25em;
-    padding-bottom: 1.25em;
-    background-color: rgba(255, 255, 255, 0.95);
-    box-shadow: 0 4px 4px rgba(0, 0, 0, 0.15);
-    backdrop-filter: blur(4px);
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  padding-top: 1.25em;
+  padding-bottom: 1.25em;
+  background-color: rgba(255, 255, 255, 0.95);
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.15);
+  backdrop-filter: blur(4px);
 }
 
 .header.scrolled .header-content {
-    padding-top: 0;
+  padding-top: 0;
 }
 
 .header.scrolled .nav a {
-    color: #48c1fb;
+  color: #48c1fb;
 }
 
 .header.scrolled .select-label {
-    color: #48c1fb;
+  color: #48c1fb;
 }
 
 .header.scrolled .new-btn {
-    border: 1px solid #ff8657;
-    color: #ff8657;
+  border: 1px solid #ff8657;
+  color: #ff8657;
 }
 
 .header.scrolled .new-btn:hover {
-    background-color: #ff8657;
-    color: #fff;
+  background-color: #ff8657;
+  color: #fff;
 }
 
 /* How it works */
 
 .method-screen {
-    padding-top: 6.25em;
-    padding-bottom: 5.5em;
-    max-width: 70em;
-    width: 100%;
-    margin: auto;
+  padding-top: 6.25em;
+  padding-bottom: 5.5em;
+  max-width: 70em;
+  width: 100%;
+  margin: auto;
 }
 
 .method-screen h1 {
-    font-weight: 500;
-    font-size: 3.75em;
-    line-height: 1.4166em;
-    margin-bottom: 1em;
-    text-align: center;
+  font-weight: 500;
+  font-size: 3.75em;
+  line-height: 1.4166em;
+  margin-bottom: 1em;
+  text-align: center;
+  font-weight: 400;
```

Comment:
If you're adding a `font-weight`, remove the duplicate `font-weight` that is overriden (line 400).